ingestion/src/metadata/ingestion/ometa/mixins/es_mixin.py

```diff
@@ -469,3 +479,25 @@ def yield_es_view_def(
                         schema_name=schema_name,
                         table_name=table_name,
                     )
+
+    def get_table_entities_from_es(
```

Comment:
nit - should we just have a generic "search_in_any_service" kind of method? so we keep the same API of passing the entity type, plus the intent is way more clear